val/src/avs_pcie.c

```diff
@@ -1888,7 +1888,7 @@ val_pcie_get_rootport(uint32_t bdf, uint32_t *rp_bdf)
   }
 
   /* Return failure */
-  val_print(AVS_PRINT_ERR, "\n       PCIe Hierarchy fail: RP of bdf 0x%x not found", bdf);
+  val_print(AVS_PRINT_ERR, "\n       PCIe Hierarchy fail: RP of bdf 0x%x not found\n", bdf);
```

Comment:
I think in keeping the existing theme of having \n at the beginning. i would suggest adding a \n to the beginning of 
val_print(AVS_PRINT_ERR, "Create Bdf table failed.\n", 0); 
instead of at the end of the line.

I do see an additional three prints in the avs_pcie.c file which are not following this convention causing this discrepancy.